### ISSUE TITLE
Clarifications on augmentation

### ISSUE THREAD

**Issue by carloalbertobarbano** [grayhong/bias-contrastive-learning#1]
Hi and thanks for your very interesting work!

I have a couple questions regarding the transformation for the contrastive loss. 

- In your paper you mention `we first construct 2N size "multiviewed batch" by applying a pair of randomized augmentation (e.g., random crop, random flip) on N data samples in a single batch`. Looking at Eqs. 1, 2 and 3 I understand that the same starting data (batch of N) is used for cross entropy and then augmented to 2N and used for contrastive loss. However I do not understand why you use a different dataloader for the contrastive loss https://github.com/grayhong/bias-contrastive-learning/blob/35562b6d700356c3a3c7346781f0310ca38f6fd1/train_biased_mnist_bc.py#L157 Wouldn't that result in effectively loading more data than the original N samples? 

- Why do you apply hue jitter on Biased-MNIST? https://github.com/grayhong/bias-contrastive-learning/blob/35562b6d700356c3a3c7346781f0310ca38f6fd1/debias/datasets/biased_mnist.py#L233

Thanks in advance!
